### ISSUE TITLE
security-context package uses runtime file loading

### ISSUE THREAD

**Issue by SamHellawell** [digitalbazaar/jsonld-signatures#165]
Would be great if this library could migrate away from the `security-context` package. I have ran into problems deploying a simple API that uses jsonld-signatures because of this package. Vercel is serverless. This is possible to solve, but is a bit of a pain.

Interestingly, the security-context has a `browser.js` file which uses requiring of json files instead of the file system. Perhaps that import could be used

**Comment by SamHellawell** [comment]
For anyone that experiences this issue or similar, I have pushed a package to NPM that doesnt use file reading: Simply add this to your package.json to override the security-context dependancy:

```
  "resolutions": {
    "security-context": "https://registry.npmjs.org/@docknetwork/security-context/-/security-context-4.0.1-0.tgz"
  },
```

Hopefully this can be fixed properly in jsonld-signatures since it doesnt look like the security-contexts repo will be updated anytime soon

**Comment by dmitrizagidulin** [comment]
Fwiw, the fix to this problem here in the `jsonld-signatures` lib is just to update the `security-context` dependency.
Meaning, upgrade from `"security-context": "^4.0.0"` to `"@digitalbazaar/security-context": "^1.0.0"`, which gets rid of runtime file loading.

**Comment by dmitrizagidulin** [comment]
Oh, in fact, I already made a PR to that effect :) https://github.com/digitalbazaar/jsonld-signatures/pull/142

**Comment by dlongley** [comment]
Addressed by #172.
